docs/update_and_migration/from_3.3/update_from_3.3.md

```diff
@@ -18,7 +18,15 @@ Go through the following steps to update to the latest maintenance release of v3
 
     If you are using v3.3.15 or earlier v3.3 version, or encounter an error related to flex.ibexa.co, you need to [update your Flex server](#update-flex-server) first.
 
-Run:
+To avoid deprecations when updating from an older PHP version to PHP 8.2 or 8.3, run the following commands:
```

Comment:
My bad, we need to move this section below the `composer require ibexa/<edition>:[[= latest_tag_3_3 =]] --with-all-dependencies --no-scripts` one - first the dependencies need to be installed